Spinner/Spinner.swift

```diff
@@ -84,25 +84,23 @@ public class SpinnerView: NSObject, Spinner {
         let view = showSpinner(inView: button, style: style, color: color)
         
         if let spinnerView = view as? SpinnerView {
-            spinnerView.userInteractionEnabledAtReception = button.isUserInteractionEnabled
```

Comment:
Its setting it twice, it sets it when you set the spinner in the showSpinner(inView:_ call. Because of this, it was causing the buttons user interaction to be disabled after the spinner was dismissed even though when the spinner was first shown, interaction was enabled.